sdk/lib/drivers/sound/mmebuddy/wave/streaming.c

```diff
@@ -206,6 +212,16 @@ CompleteIO(
             /* Partially completed */
             HdrExtension->BytesCompleted += dwNumberOfBytesTransferred;
             SND_TRACE(L"%d/%d bytes of wavehdr completed\n", HdrExtension->BytesCompleted, WaveHdr->dwBufferLength);
+
+            /* Do we loop a header? */
+            if (HdrExtension->BytesCommitted == WaveHdr->dwBufferLength &&
+                SoundOverlapped->PerformCompletion == FALSE)
+            {
+                /* Reset amount of bytes and decrement loop count,
+                 * to play next iteration */
```

Comment:
```suggestion
                /* Reset amount of bytes and decrement loop count, to play next iteration */
```
since the SND_TRACE at line 214 is even longer than that.

Reply:
Committed, thanks.